content/blog/2017/2017-02-14-declarative-notifications.adoc

```diff
@@ -0,0 +1,436 @@
+---
+layout: post
+title: "Declarative Pipeline: Notifications and Shared Libraries"
+tags:
+- tutorial
+- pipeline
+- declarative
+- plugins
+- notifications
+- slack
+- hipchat
+- email-ext
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at link:https://cloudbees.com[CloudBees].
+
+**Declare Your Pipelines!**
+link:/blog/2017/02/03/declarative-pipeline-ga/[Declarative Pipeline 1.0 is here]!
+This is the third post in a series showing some of the cool features of
+link:/doc/book/pipeline/syntax/#declarative-pipeline[Declarative Pipeline].
+
+
+In the
+link:/blog/2017/02/10/declarative-html-publisher/[previous blog post],
+we converted Scripted Pipeline to a Declarative Pipeline with descriptive stages
+and `post` blocks.  In one of those `post&#123; }`  blocks, we included a placeholder for
```

Comment:
post *section*, post *section*. Please be consistent with phrasing used in the [syntax reference](https://jenkins.io/doc/book/pipeline/syntax/)